### PR TITLE
Reqs TGUI

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
Reqs tablet sprite from #3607
image

## Why It's Good For The Game
makes playing reqs less painful

## Changelog
:cl:
add: Reqs computer now uses tgui
add: RO now has a wireless reqs tablet
add: Requests computer now uses tgui
add: Orders are now multiple supply packs and come in a single crate
tweak: Many supply packs have been changed to be smaller to make it easier to only order exactly what you need.
/:cl:

Closes #3607

### CODE REVIEW

**Comment by spookydonut** [comment]
Needs some tgui cleanup

**Comment by spookydonut** [comment]
image

**Comment by spookydonut** [comment]
image

**Comment by InterroLouis** [comment]
The requests console breaks if you go to the 'view requests' page because there's no back button.

**Comment by spookydonut** [comment]
> 
> 
> The requests console breaks if you go to the 'view requests' page because there's no back button.
> Just for more clarification I mean the ordering console outside of req.

yeah i'll be turning that into tgui next, assuming thats caused by a runtime

**Comment by Pariah919** [comment]
mind doing this one #4076 is merged

/datum/supply_packs/T19_holster->remove the entire pack

/datum/supply_packs/webbing->remove the t19 holster in it

/datum/supply_packs/stock_shotgun->remove the entire pack

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -38,2607 +27,1468 @@ GLOBAL_LIST_INIT(all_supply_groups, list("Operations", "Weapons", "Hardpoint Mod
...
+	contains = list(/obj/item/ammo_magazine/rocket/wp)
+	cost = 5
+
+/datum/supply_packs/ammo/smartgun
+	name = "M56 smartgun powerpack"
+	contains = list(/obj/item/smartgun_powerpack)
+	cost = 12
+
+/datum/supply_packs/ammo/smartmachinegun
+	name = "T26 smartmachinegun ammo"
```
Comment (InterroLouis) [inline code/datums/supplypacks.dm:695]: name = "T29 smartmachinegun ammo"

Comment (spookydonut) [reply code/datums/supplypacks.dm:695]: i guess whoever added it put the wrong numbers in

File: code/modules/shuttle/supply.dm [hunk 2]
```diff
@@ -149,64 +148,70 @@ GLOBAL_LIST_EMPTY(exports_types)
...
 
-
+	// TODO: fix this
 	for(var/place in shuttle_areas)
 		var/area/shuttle/shuttle_area = place
 		for(var/atom/movable/AM in shuttle_area)
 			if(AM.anchored)
 				continue
-
+			/*
```
Comment (psykzz) [inline code/modules/shuttle/supply.dm:214]: Just delete it, if we want more points later we can bring it back another way.

File: code/modules/shuttle/supply.dm [hunk 3]
```diff
@@ -231,365 +239,338 @@ GLOBAL_LIST_EMPTY(exports_types)
...
+				SSshuttle.moveShuttle("supply", "supply_home", TRUE)
+			. = TRUE
+		if("approve")
+			var/datum/supply_order/O = SSpoints.requestlist["[params["id"]]"]
+			if(!O)
+				O = SSpoints.deniedrequests["[params["id"]]"]
+			if(!O)
+				return
+			SSpoints.approve_request(O, ui.user)
+			. = TRUE
```
Comment (psykzz) [inline code/modules/shuttle/supply.dm:472]: Does TGUI handle validation of the request?
How do we stop some random href validating these?

**Comment by XElectricX** [comment]
AI player reported they could not interact with the console.

**Pull request merged**